src/platform/APM32/mk/APM32F4.mk

```diff
@@ -206,7 +204,7 @@ MSC_SRC = \
             msc/usbd_storage_sdio.c
 
 SPEED_OPTIMISED_SRC += \
-            common/stm32/system.c
+            stm32/system.c
```

Comment:
(it takes time to find out that this is src/platform/common. And there are both STM32 and stm32 directories, with handling different between windows and other systems)..

Reply:
I have made significant changes to remove VPATH entries.